usermods/smartnest/usermod_smartnest.h

```diff
@@ -1,14 +1,14 @@
-#ifndef WLED_ENABLE_MQTT
-#error "This user mod requires MQTT to be enabled."
-#endif
-
 #pragma once
 
 #include "wled.h"
 
 class Smartnest : public Usermod
 {
 private:
+  bool initialized = false;
+  unsigned long lastMqttReport = 0;
+  const unsigned long mqttReportInterval = 60000; // Report every minute
```

Comment:
There  is really no need for this to be `unsigned long` and being `const` prevents this to be user adjustable via configuration.